src/NoMatch.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import './NoMatch.css'
 
 function NoMatch() {
   return (
-    <section>
-      <p>No page found.</p>
+    <section className='no-page'>
+      <p>404 No page found.</p>
```

Comment:
Love this. Let's plan to add more to this in the future, butI do like it and how it functions!